homework/src/index.js

```diff
@@ -16,6 +15,98 @@
     xhr.send();
   }
 
+  function repoData(error, data) {
+    if (error !== null) {
+      console.error(error);
+    } else {
+      const arrayOfObjects = JSON.parse(data);
+      const root = document.getElementById('root');
+
+      const header = createAndAppend('div', root);
+      header.id = 'sub_root';
+
+      const heading = createAndAppend('h1', header);
+      heading.innerHTML = 'HYF Repositories :';
+      const select = createAndAppend('select', header);
+      select.setAttribute('class', 'select_menu');
+      const optionItem = createAndAppend('option', select);
+      optionItem.innerText = 'Choose a repository';
+      for (const obj of arrayOfObjects) {
+        const optionItem = createAndAppend('option', select);
+        optionItem.innerText = obj.name;
+        optionItem.value = 'https://api.github.com/repos/HackYourFuture/' + obj.name;
+      }
+      select.addEventListener('change', request => {
+        const newUrl = event.target.value;
+        const root = document.getElementById('root');
+
+        root.innerHTML = null;
+        fetchJSON("https://api.github.com/orgs/HackYourFuture/repos?per_page=100", repoData);
```

Comment:
You should not use URLs like this, you should put them in a variable instead. Without a variable, it can get confusing when you try to reuse the code or try to change something